pkg/nodeconfig/payload/payload.go

```diff
@@ -81,6 +81,12 @@ const (
 	//       related. Correct that in: https://issues.redhat.com/browse/WINC-882
 	// networkConfTemplate is the template used to generate the network configuration script
 	networkConfTemplate = `# This script ensures the contents of the CNI config file is correct, and returns the HNS endpoint IP
+
+param(
+    [string]$NodeName,
+    [string]$NodeSubnet
```

Comment:
consider naming the input params according to the usage, i.e. 

```
    [string]$hostnameOverride,
    [string]$clusterCIDR
```


the    mapping should happened where the invocation takes place